llumnix/llm_engine_manager.py

```diff
@@ -227,10 +231,14 @@ async def _push_migrations(self) -> None:
 
     async def _migrate(self, pair_migration_type: PairMigrationConstraints, migrate_in_num_requests: int) -> None:
         async def migrate_done_callback(ret, migrate_instance_pair: Tuple[str, str]) -> None:
-            if migrate_instance_pair[0] in self.instance_migrating:
-                self.instance_migrating[migrate_instance_pair[0]] = False
-            if migrate_instance_pair[1] in self.instance_migrating:
-                self.instance_migrating[migrate_instance_pair[1]] = False
+            if pair_migration_type != PairMigrationConstraints.PREFILL_2_DECODING:
```

Comment:
In your PR, does the instance can act as both a migration source and a migration target? Given that, why exclude the P2D? Since in PDD, a decoding instance can act as a migrate-out end (D->D) while acting as a migrate-in end (P->D).

Reply:
Now, self.instance_migrating is deprecated because the vast majority of migrations can be completed within a single update_instance_interval. Therefore, it is unnecessary to track the migration status for the purpose of calculating load and adjusting migration strategies.